src/scss/grommet-core/_objects.card.scss

```diff
@@ -53,6 +53,10 @@ $card-hover-color: #EBEBEB;
 }
 
 .#{$grommet-namespace}card__content {
+  .#{$grommet-namespace}tiles & {
+    flex-grow: 1;
```

Comment:
As part of my style modularization work, I'm trying to avoid selectors that cross components like this. Instead, can we just put `flex="grow"` on the Box used for the card content?